src/Integration.Vsix/CFamily/CFamilyStrings.resx

```diff
@@ -118,7 +118,10 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="ERROR_Analysis_Failed" xml:space="preserve">
-    <value>Failed to execute analysis, exception details: {0}</value>
+    <value>Failed to analyze {0}, exception details: {1}</value>
```

Comment:
I noticed that the "Failed" message didn't have the file name. I thought it was useful information to have.